### ISSUE TITLE
[request] Doubleclick on preview window canvas to make it full screen

### ISSUE THREAD

**Issue by bencresty** [EsotericSoftware/spine-editor#682]
Currently busy creating an animation consisting of multiple animations I mix these in the preview to view them all in action together. This will be the final result.

Because this mixing can only be done in the preview of the editor we're now missing the preview function (F9 on Windows, which is great btw!!). 

It would be great if we could make the preview window full screen by double clicking on the preview canvas so we can better view the final result while animating!

**Comment by NathanSweet** [comment]
I agree that maximizing a view would be useful. We'll have to consider the details of how it'll work, as we don't currently always have some space dedicated to the viewport area.
